homeassistant/components/nederlandse_spoorwegen/strings.json

```
@@ -0,0 +1,144 @@
+        "data": {
+          "name": "Route name",
+          "from": "From station",
+          "to": "To station",
+          "via": "Via station (optional)",
+          "time": "Departure time (optional)"
+        },
+        "data_description": {
+          "name": "A name for this route.",
+          "from": "Departure station code (e.g., AMS)",
+          "to": "Arrival station code (e.g., UTR)",
+          "via": "Optional via station code (e.g., RTD)",
+          "time": "Optional departure time in HH:MM:SS (24h)"
+        }
+      },
+      "reauth": {
+        "title": "Re-authenticate",
+        "description": "Your NS API key needs to be updated.",
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]"
+        },
+        "data_description": {
+          "api_key": "Enter your new NS API key."
+        }
+      },
+      "reconfigure": {
+        "title": "Reconfigure",
+        "description": "Update your NS API key.",
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]"
+        },
+        "data_description": {
+          "api_key": "Enter your new NS API key."
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "Could not connect to NS API. Check your API key.",
+      "invalid_auth": "Invalid API key.",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "This API key is already configured.",
+      "reauth_successful": "Re-authentication successful.",
+      "reconfigure_successful": "Reconfiguration successful."
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "title": "Configure routes",
+        "description": "Configure the routes.",
+        "data": {
+          "action": "Action"
+        },
+        "data_description": {
+          "action": "Edit, add, or delete train routes for this integration."
+        }
+      },
+      "select_route": {
+        "title": "Select route",
+        "description": "Choose a route to {action}.",
+        "data": {
+          "route_idx": "Route"
+        },
+        "data_description": {
+          "route_idx": "Select the route you want to edit or delete."
+        }
+      },
+      "add_route": {
+        "title": "Add route",
+        "description": "[%key:component::nederlandse_spoorwegen::config::step::routes::title%]",
+        "data": {
+          "name": "Route name",
+          "from": "From station",
+          "to": "To station",
+          "via": "Via station (optional)",
+          "time": "Departure time (optional)"
+        },
+        "data_description": {
+          "name": "A name for this route.",
+          "from": "Departure station code (e.g., AMS)",
+          "to": "Arrival station code (e.g., UTR)",
+          "via": "Optional via station code (e.g., RTD)",
+          "time": "Optional departure time in HH:MM:SS (24h)"
```

Let's deduplicate all those, too:

```suggestion
          "name": "[%key:component::nederlandse_spoorwegen::config::step::routes::data::name%]",
          "from": "[%key:component::nederlandse_spoorwegen::config::step::routes::data::from%]",
          "to": "[%key:component::nederlandse_spoorwegen::config::step::routes::data::to%]",
          "via": "[%key:component::nederlandse_spoorwegen::config::step::routes::data::via%]",
          "time": "[%key:component::nederlandse_spoorwegen::config::step::routes::data::time%]"
        },
        "data_description": {
          "name": "[%key:component::nederlandse_spoorwegen::config::step::routes::data_description::name%]",
          "from": "[%key:component::nederlandse_spoorwegen::config::step::routes::data_description::from%]",
          "to": "[%key:component::nederlandse_spoorwegen::config::step::routes::data_description::to%]",
          "via": "[%key:component::nederlandse_spoorwegen::config::step::routes::data_description::via%]",
          "time": "[%key:component::nederlandse_spoorwegen::config::step::routes::data_description::time%]"
```

Same for that identical block below.